ref/src/sw_core.F90

```diff
@@ -126,8 +126,8 @@ subroutine c_sw(sw_corner, se_corner, nw_corner, ne_corner,       &
 
     if ( nord > 0 ) then
       call divergence_corner(sw_corner, se_corner, ne_corner, nw_corner, &
-                             rarea_c, sin_sg, cos_sg, cosa_u, cosa_v,    &
```

Comment:
I thought this was just removing #ifdef(s).  This is a change to an interface.

Reply:
The variables removed form the interface were only used in the IFDEF'd part of the code so when the IFDEFs were removed those variables resulted in a compiler warning that they were never referenced so I thought it would be cleaner to remove the variables.

Reply:
@LyndStringer - I also noticed the change in interface, but concluded that it was correct since those variables are no longer used in that subroutine.  This boils down to the same question that is motivating these changes in the first place.  We are removing dead (for our use case) code.  If we've decided that we're only going to keep what is actually on the code path our use case follows, then I think this cleanup fits that goal.  If we start saying we'll keep "this" dead code but throw away "that" dead code, then it's hard to know what the guiding principle/goal is.  I say remove it.

Reply:
Since Lynd agrees, this issue is resolved by leaving the questioned changes as they were.

Reply:
My bad, I thought Jacques could resolve this conversation.  I'll resolve it immediately.